src/jit/function-builder.cc

```diff
@@ -195,14 +198,43 @@ bool FunctionBuilder::buildIL() {
   return true;
 }
 
-void FunctionBuilder::SetUpLocals(TR::IlBuilder* b, const uint8_t** pc, VirtualStack* stack) {
+bool FunctionBuilder::SetUpLocals(TR::IlBuilder* b, const uint8_t** pc, VirtualStack* stack) {
   // Add placeholders onto the virtual stack. These values should never be actually read, but will
   // eventually be dropped using drop or drop_keep in the function epilogue.
+  static const auto supported_types = std::unordered_set<Type>({Type::I32, Type::I64, Type::F32, Type::F64});
```

Comment:
There isn't a default implementation of `std::hash` for enumeration types until C++14, so this won't compile correctly using only C++11.